### PR TITLE
Fixed focus on monterrey

### PR DESCRIPTION
**Pull request by Fiser12**
I recommend you this change in your PR. It's for distinguish between different kind of Focus states

### CODE REVIEW

File: lib/dnd/monterey-macos-dnd.h [hunk 1]
```diff
@@ -4,6 +4,7 @@
 + (bool)isEnabled;
 + (bool)enabledByAssertion;
 + (bool)enabledBySchedule;
++ (NSString*)getFocusMode;
```
Comment (nbashkankov) [inline lib/dnd/monterey-macos-dnd.h]: Maybe rename it to getActiveFocusMode ?

File: lib/dnd/monterey-macos-dnd.mm [hunk 2]
```diff
@@ -10,6 +10,7 @@ + (bool)isEnabled {
 
   // 1. check assertion
   bool isDNDEnabled = [self enabledByAssertion];
```
Comment (nbashkankov) [inline lib/dnd/monterey-macos-dnd.mm]: looks like my initial naming not accurate - maybe rename it to "isFocusModeEnabled" or maybe you have better name

**Review by nbashkankov — COMMENTED** [review]
Overall, thank you for help. Mine changes cover only built-in "DND" focus mode.
With your changes now it cover additional "user-defined" focus modes.
I am not insist on changes related to my comments. 
Just reply if you not going to fix them. 
After that i will merge your changes.

**Comment by Fiser12** [comment]
I've added your suggestions to my PR, thanks!

**Comment by Fiser12** [comment]
BTW I've tested it on an Intel Macbook with Monterrey and on a M1 Macbook with Big Sur, both works like a charm. I don't have previous macOS version to test it.

**Comment by nbashkankov** [comment]
Thank you!

**Pull request merged**